src/features/payment/components/PaymentNoticeChoice/PaymentNoticeChoice.tsx

```diff
@@ -2,19 +2,42 @@ import ArrowForwardIosIcon from "@mui/icons-material/ArrowForwardIos";
 import KeyboardIcon from "@mui/icons-material/Keyboard";
 import QrCodeScannerIcon from "@mui/icons-material/QrCodeScanner";
 import { Box, Grid, Typography } from "@mui/material";
+import mixpanel from "mixpanel-browser";
 import React from "react";
 import { useTranslation } from "react-i18next";
 import { useNavigate } from "react-router";
+import {
+  CHK_PAYMENT_NOTICE_DATA_ENTRY,
+  CHK_PAYMENT_NOTICE_DATA_ENTRY_MANUAL,
+  CHK_PAYMENT_NOTICE_QRCODE_SCAN,
+} from "../../../../utils/mixpanel/mixpanelEvents";
 import { CheckoutRoutes } from "../../../../routes/models/routeModel";
 
 export function PaymentNoticeChoice() {
   const { t } = useTranslation();
   const navigate = useNavigate();
 
+  React.useEffect(() => {
+    mixpanel.track(CHK_PAYMENT_NOTICE_DATA_ENTRY.value, {
+      EVENT_ID: CHK_PAYMENT_NOTICE_DATA_ENTRY.value,
+    });
+  }, []);
+
   const handleClickOnQR = React.useCallback(() => {
+    sessionStorage.setItem("notice_code_data_entry", "qr_code");
+    mixpanel.track(CHK_PAYMENT_NOTICE_QRCODE_SCAN.value, {
+      EVENT_ID: CHK_PAYMENT_NOTICE_QRCODE_SCAN.value,
+      notice_code_data_entry: "qr_code",
```

Comment:
```suggestion
      data_entry: "qr_code",
```
notice_code_data_entry only on sessionStorage

Reply:
Solved by https://github.com/pagopa/pagopa-checkout-fe/pull/518/commits/d9b8cbea72d29640370eb3673aa2148c12be0209